lib/exercises.js

```diff
@@ -43,12 +43,37 @@ Exercise.prototype.addExercises = function () {
 };
 
 Exercise.prototype.updateExercise = function () {
+  var that = this;
   $('.exercise-name, .exercise-calories').off('click');
   var $exerciseRow = $(this).parent();
   var $name = $exerciseRow.children('.exercise-name');
-  $name.html(`<input class='create-exercise-name' type="text" name='name' value=${$name.text()}></br>`)
+  var oldName = $name.text();
+  $name.html(`<input class='update-exercise-name' type="text" name='name' value=${$name.text()}></br>`)
   var $calories = $exerciseRow.children('.exercise-calories');
-  $calories.html(`<input class='create-exercise-calorie-count' type="text" name='calories' value=${$calories.text()}></br>`)
+  $calories.html(`<input class='update-exercise-calorie-count' type="text" name='calories' value=${$calories.text()}></br>`)
+  $('.update-exercise-name, .update-exercise-calorie-count').on('blur', function () {
+    var name = $('input.update-exercise-name').val();
+    var calories = $('input.update-exercise-calorie-count').val();
+    var newExercise = new Exercise;
+    newExercise.updateLocal(oldName, name, calories);
+  });
+};
+
+Exercise.prototype.updateLocal = function (oldName, name, calories) {
+  var exercisesJSON = localStorage.getItem('exercises');
```

Comment:
This set of lines pulling from local storage and then updating from local storage gets repeated a lot. We should extract this when we refactor.